<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>320</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>426</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>426</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Setting</string>
  </property>
  <widget class="QLabel" name="lblTitle1">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>8</y>
     <width>185</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Skin Color Detection Setting</string>
   </property>
  </widget>
  <widget class="QLabel" name="hueLabel">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>36</y>
     <width>25</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Hue</string>
   </property>
  </widget>
  <widget class="QLabel" name="saturationLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>36</y>
     <width>63</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Saturation</string>
   </property>
  </widget>
  <widget class="QLabel" name="valueLabel">
   <property name="geometry">
    <rect>
     <x>332</x>
     <y>32</y>
     <width>34</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Value</string>
   </property>
  </widget>
  <widget class="QSlider" name="barMinH">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>60</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="barMaxH">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>88</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="barMinS">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>60</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="barMaxS">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>88</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="barMinV">
   <property name="geometry">
    <rect>
     <x>296</x>
     <y>60</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="barMaxV">
   <property name="geometry">
    <rect>
     <x>296</x>
     <y>88</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReset">
   <property name="geometry">
    <rect>
     <x>296</x>
     <y>270</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMinH">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>76</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMaxH">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>108</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMaxS">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>108</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMaxV">
   <property name="geometry">
    <rect>
     <x>372</x>
     <y>108</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMinV">
   <property name="geometry">
    <rect>
     <x>372</x>
     <y>76</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMinS">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>76</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitle2">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>150</y>
     <width>185</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Detection Region</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblHorizon">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>180</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Horizon</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblVertical">
   <property name="geometry">
    <rect>
     <x>196</x>
     <y>180</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Vertical</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblFrom">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>218</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>To</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTo">
   <property name="geometry">
    <rect>
     <x>204</x>
     <y>218</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>To</string>
   </property>
  </widget>
  <widget class="QSlider" name="barMinHorizon">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>200</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="barMinVertical">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>200</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="barMaxVertical">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>230</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="barMaxHorizon">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>230</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lblMinHorizon">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>215</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0%</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMaxHorizon">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>246</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0%</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMinVertical">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>215</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0%</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMaxVertical">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>246</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0%</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>138</y>
     <width>394</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lblDetectionArea">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>97</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Detection Area</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="boxDetectionArea">
   <property name="geometry">
    <rect>
     <x>331</x>
     <y>172</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QLabel" name="lblActionInterval">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>212</y>
     <width>129</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Action Interval</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="boxInterval">
   <property name="geometry">
    <rect>
     <x>331</x>
     <y>236</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <number>250</number>
   </property>
   <property name="maximum">
    <number>2500</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
